.github/workflows/ci.yml

```diff
@@ -46,7 +52,8 @@ jobs:
         run: ./mill -k --disable-ticker __.mimaReportBinaryIssues
 
       - name: Publish ${{ github.ref }}
-        if: startsWith(github.ref, 'refs/tags/v')
+        if: startsWith(github.ref, 'refs/tags/v') ||
+          (github.event_name == 'workflow_dispatch' && github.event.inputs.publishSnapshot == 'true')
```

Comment:
just making sure: do you know what version is generated? is there a plugin like dynver in here?

Reply:
Yeah, if you run `mill show __.publishVersion` you can see it (note you need to export `CI` env var if running locally or else you'll get `dev-SNAPSHOT`)

Reply:
For example:
![CleanShot 2024-01-30 at 10 39 55](https://github.com/disneystreaming/alloy/assets/13733772/4a8220b0-5da1-4068-add5-454f24a4dcdb)